app/javascript/react/screens/App/Plan/components/PlanRequestDetailList/PlanRequestDetailList.js

```diff
@@ -639,27 +641,41 @@ class PlanRequestDetailList extends React.Component {
                     </ListView.InfoItem>
                   ]}
                   actions={
-                    downloadLogInProgressTaskIds &&
-                    downloadLogInProgressTaskIds.find(element => element === task.id) ? (
-                      <label htmlFor="downloadLog">
-                        <span id="downloadLogInProgress">{__('Download log in progress...')}</span>
-                      </label>
-                    ) : (
-                      <DropdownButton
-                        id={`${task.id}-${task.descriptionPrefix}_download_log_dropdown`}
-                        title={__('Download Log')}
-                        pullRight
-                        onSelect={eventKey => this.onSelect(eventKey, task)}
+                    <DropdownButton
+                      id={`${task.id}-${task.descriptionPrefix}_download_log_dropdown`}
+                      title={__('Download Log')}
+                      pullRight
+                      onSelect={eventKey => this.onSelect(eventKey, task)}
+                      disabled={
+                        downloadLogInProgressTaskIds &&
+                        downloadLogInProgressTaskIds.find(element => element === task.id) &&
+                        !task.options.prePlaybookComplete &&
+                        !task.options.postPlaybookComplete
+                      }
+                    >
+                      {task.options.prePlaybookComplete && (
+                        <MenuItem
+                          eventKey="preMigration"
+                          disabled={downloadLogInProgressTaskIds && downloadLogInProgressTaskIds.indexOf(task.id) > -1}
```

Comment:
The dropdown button is disabled only if 
```js
disabled={
  downloadLogInProgressTaskIds &&
  downloadLogInProgressTaskIds.find(element => element === task.id) &&
  !task.options.prePlaybookComplete &&
  !task.options.postPlaybookComplete
 }
```

So if `task.options.prePlaybookComplete` or `task.options.postPlaybookComplete` exists, the dropdown button would need to be clickable (right?), and then we have to disable the individual menu items.

Reply:
Ahh ok.  So then we would allow downloading the migration log simultaneously with, say the pre-playbook log.

Right, ok, that makes sense.  Sorry, this is challenging to review without a good way to test these particular scenarios

Reply:
I tested it locally with some trickery -- so agree that it is challenging to review unless you test it.